nucypher/cli/commands/deploy.py

```diff
@@ -336,7 +371,8 @@ def upgrade(general_config, actor_options, retarget, target_address, ignore_depl
                                                                     target_address=target_address), abort=True)
         transaction = ADMINISTRATOR.retarget_proxy(contract_name=contract_name,
                                                    target_address=target_address,
-                                                   just_build_transaction=True)
+                                                   just_build_transaction=True,
+                                                   confirmations=confirmations)
```

Comment:
The multisig workflow (which is not being used, to be fair) doesn't use confirmations, as it only builds a TX call data, so the changes in these 2 lines should be reverted.

Reply:
I see - I added the `confirmations` kwargs here because it's now required.  In order to exclude it here, Ill need to make it optional.

Reply:
To be fair, it's just going to be ignored, so it's ok to leave it as is, then.